src/components/jacobian_component/make_perturbation_sf.py

```diff
@@ -124,6 +124,14 @@ def calculate_sfs(state_vector, emis_prior, target_emission=1e-8, prior_sf=None)
 
     # calculate the effective scale factors
     effective_pert_sf = jacobian_pert_sf / flat_prior_sf
+    
+    # subtract 1.0 from the effective scale factors to get the percent 
+    # change used in the inversion sensitivity calculations
+    # eg. 50% == 0.5
+    # This is is different from the jacobian simulations
+    # where we apply the "jacobian_pert_sf" in HEMCO.
+    # 50% perturbation is represented by SF of 1.5
```

Comment:
Could you be more precise with the terms "perturbation" and "SF"? It's unclear from this comment how to interpret the code on the next line since the term "sf" or "SF" is used in multiple ways in the code and in the comment